src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #e5e5e5;
+  display: flex;
+  flex-direction: column;
```

Comment:
do you even need flex here? block elements will stack in column by default
```suggestion
```

Reply:
done